lib/apiservers/engine/backends/container.go

```diff
@@ -151,7 +151,7 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 	var layer *viccontainer.VicContainer
 	container := viccontainer.GetCache().GetContainerByName(config.Config.Image)
 	if container == nil {
-		layer, err = c.getImageMetadataFromStoragePL(config.Config.Image)
+		layer, err = c.getImageMetadataFromImageC(config.Config.Image)
```

Comment:
see above.  I was in that file when I started inspect and noticed the function was incorrectly named.  It's not related to the purpose of the PR.